samplib/ZWESLSTC

```diff
@@ -57,5 +57,5 @@
 _CEE_ENVFILE_CONTINUATION=\
 _CEE_RUNOPTS=HEAPPOOLS(OFF)
```

Comment:
Should we update this line to contain HEAPPOOLS64(OFF) or change it when we detect `components.zss.agent.64bit=true`?
I am asking and I don't know the answer, but it helped [here](https://github.com/zowe/zss/issues/685#issuecomment-2004153640).

Reply:
I think it should be done unconditionally because i think configmgr itself is 64 bit.